custom_components/moonraker/__init__.py

```diff
@@ -109,10 +109,16 @@ async def _async_update_data(self):
             METHOD.PRINTER_OBJECTS_QUERY, self.query_obj
         )
         info = await self.async_fetch_data(METHOD.PRINTER_INFO)
+        history = await self.async_fetch_data(METHOD.SERVER_HISTORY_TOTALS)
```

Comment:
In case there is no history available, this will return a "Method not found" error every time, not an exception but history will me something like {"error": "method not found"}. 

Any suggestion ?